### ISSUE TITLE
Navigation Tree/Structure Panel

### ISSUE THREAD

**Issue by codente** [beaverbuilder/feature-requests#1]
A panel so that you can see the structure of a page and possibly move things around from there as well

**Comment by Alok-S-Sharma** [comment]
I would love to see this (must have) feature as I often get lost and have a difficult time finding/arranging the modules, rows & columns.

**Comment by gbissland** [comment]
@Alok-S-Sharma You can download the new 1.4 alpha/beta from your account page and try this feature now. It is working well!

**Comment by codente** [comment]
> @Alok-S-Sharma You can download the new 1.4 alpha/beta from your account page and try this feature now. It is working well!

It's BB 2.5 Alpha.  1.4 is Themer Alpha and does not have the Outline Panel.   Beta release is imminent as well!

**Comment by Alok-S-Sharma** [comment]
Thanks @gbissland & @codente 

I have installed the alpha & beta release in my sandbox website, and the Navigation Tree/Structure is very much working as desired. Actually, it has made my life much easier. 

BTW, as I was clearing the BB cache, I realized there is an option to enable Beta & Alpha releases. I have checked them as well to enable auto-updates.

Will provide the necessary feedback if I encounter any problems.
